data/src/languages_update.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+"""languages.json update
+
+Takes languages.json file and updates the script entry for a language based on
+the files that are present in data/tsv.
+"""
+import collections
+import json
+import operator
+import os
+import unicodedataplus  # type: ignore
+
+from typing import Dict, DefaultDict, Optional
+
+from data.src.codes import (  # type: ignore
```

Comment:
I think this can be one-line and if there's a typing problem it should be easy to fix since it's in-module.